st2common/tests/unit/test_shell_action_system_model.py

```diff
@@ -21,12 +21,17 @@
 
 from st2common.models.system.action import ShellCommandAction
 from st2common.models.system.action import ShellScriptAction
-
+from st2common.logging.formatters import MASKED_ATTRIBUTE_VALUE
```

Comment:
Import from `secrets` here as well?

Reply:
Yep, will update it.

Reply:
It looks like it's actually an in-direct import, which is kinda nasty.

I need to check why pylint didn't detect this - we have "__all__" in `st2common.logging.formatters` which doesn't include `MASKED_ATTRIBUTE_VALUE` so it should detect that.